gradle-plugin/build.gradle

```diff
@@ -23,18 +23,43 @@ gradlePlugin {
   }
 }
 
-pluginBundle {
-  website = 'https://pekko.apache.org/docs/pekko-grpc/current/'
-  vcsUrl = 'https://github.com/apache/incubator-pekko-grpc'
-  description = 'Building streaming gRPC servers and clients on top of Apache Pekko Streams'
-  tags = ['pekko', 'streams', 'reactive']
+publishing {
+    publications.withType(MavenPublication) {
+        pom {
```

Comment:
A lot of the pom details were outdated/missing. This data was retrieved by copying it from the `pom` of one of the sbt artifacts of this project